.gitignore

```diff
@@ -12,6 +12,7 @@ db.sqlite3-journal
 media
 mediafiles
 staticfiles
+src/static
```

Comment:
remove the folder and re-run `collectstatic` command to recollect everything under staticfiles (which already is gitignored)